Makefile

```diff
@@ -108,6 +108,9 @@ benchFrontendBattle:
 benchCrawl:
 	dist/build/LambdaHack/LambdaHack --dbgMsgSer --logPriority 4 --newGame 1 --noAnim --maxFps 100000 --frontendNull --benchmark --stopAfterFrames 7000 --automateAll --keepAutomated --gameMode crawl --setDungeonRng 0 --setMainRng 0
 
+benchCrawl-new-build:
+	$$(cabal-plan list-bin LambdaHack) --dbgMsgSer --logPriority 4 --newGame 1 --noAnim --maxFps 100000 --frontendNull --benchmark --stopAfterFrames 7000 --automateAll --keepAutomated --gameMode crawl --setDungeonRng "SMGen 123 123" --setMainRng "SMGen 125 125"
```

Comment:
Oh, this is cool. I will probably convert all my makefiles (and possibly CI scripts) to `cabal-plan`, though I'd need to install it in each CI and also mention in README.md as a pre-requisite for using the makefile.